modules/ROOT/pages/limitations-self.adoc

```diff
@@ -17,3 +17,13 @@ Run no more than 20 - 25 replicas per core, up to a maximum of 40 replicas per n
 | Business Groups | You can create up to 50 Runtime Fabrics per org in a Business Group. Any sub org can contain up to 50 Runtime Fabrics, in addition to any shared by another sub org. For example, if you have master Org A and its child Org B, you can have 50 Runtime Fabrics in Org A and 50 in Org B. You can also share all 50 Runtime Fabrics from Org A with Org B, and as a result, you will see 100 Runtime Fabrics in total in the list view of Org B.
 | Object Store Persistence | To manage data persistence across Mule application replicas and restarts, Anypoint Runtime Fabric uses Persistence Gateway. See xref:persistence-gateway.adoc[Persistence Gateway].
 |===
+
+== How Antivirus and DPI Software Impacts Runtime Fabric Functionality
+
+The Anypoint Runtime Fabric appliance has specific hardware and OS requirements. MuleSoft Support provides Support SLA and Severity Levels based on our validation and certification of the requirements specified in the xref:install-prereqs.adoc[installation prerequisites].
+
+A small number of third-party software that requires root access to the host, which includes, but is not limited to, antivirus and DPI software, has been found to interfere with the behavior and requirements of the Runtime Fabric appliance. In many support cases where MuleSoft Support has detected antivirus software, they have seen issues such as port blocking, node-to-node communication blocking, Docker default bridge deletion, and filesystem access issues that interfere with Runtime Fabric appliance installation, upgrade, and normal operations.
```

Comment:
Same comment as in the other PR about the verb-noun agreement and/or use of "number of ... software"